docs/4.3. Behaviour - Token Requests.md

```diff
@@ -18,7 +18,7 @@ The Client Credentials grant MUST be supported by the Authorization Server. This
 confidential clients in machine to machine use cases. Authorization Servers SHOULD provide the means to restrict which
 scopes can be used with this grant type either during dynamic client registration or during token requests.
 
-The Refresh Token Grant MUST be supported in order to ease the refreshing of access tokens and reduce the number user
+The Refresh Token Grant MUST be supported in order to ease the refreshing of Access Tokens and reduce the number user
```

Comment:
There's a mix through the documentation of whether the word 'grant' is capitalized in phrases like "Refresh Token grant" or "Refresh Token Grant".